ci/test/01_base_install.sh

```diff
@@ -6,35 +6,27 @@
 
 export LC_ALL=C.UTF-8
 
-# This script is always run as root, and the functions can be removed and
-# replaced by a bash -c "command" directly, where needed.
-# For now, they are named aliases to aid review with --color-moved=dimmed-zebra.
-CI_EXEC_ROOT () { bash -c "$*"; }
-CI_EXEC()       { bash -c "$*"; }
-export -f CI_EXEC_ROOT
-export -f CI_EXEC
```

Comment:
Aren't there other scripts relying on the exported aliases ? Won't they fail ?

Reply:
Yes, but the next time they are used in `ci/test/04_install.sh`, they are already redefined on lines 79-84:
`CI_EXEC () {
  $CI_EXEC_CMD_PREFIX bash -c "export PATH=${BINS_SCRATCH_DIR}:\$PATH && cd \"$P_CI_DIR\" && $*"
}
CI_EXEC_ROOT () {
  $CI_EXEC_CMD_PREFIX_ROOT bash -c "export PATH=${BINS_SCRATCH_DIR}:\$PATH && cd \"$P_CI_DIR\" && $*"
}`

This PR only relates to their usage in base setup.

Reply:
iirc, `./depends` was the only folder affected, so removing the second user account and running as root seems fine.